src/sentry/static/sentry/app/components/onboardingWizard/toDoItem.jsx

```diff
@@ -119,49 +118,142 @@ const TodoItem = createReactClass({
-            {task.status === 'pending' && <span className="icon-ellipsis" />}
-          </div>
-          <a href={learnMoreUrl} onClick={this.handleClick}>
-            <h4>{task.title}</h4>
-          </a>
-          <div>{description}</div>
-          {showSkipButton && (
-            <a className="skip-btn btn btn-default" onClick={this.toggleConfirmation}>
-              {t('Skip')}
-            </a>
-          )}
-        </div>
-        {showConfirmation && (
-          <Confirmation
-            task={task.task}
-            onSkip={() => this.onSkip(task.task)}
-            dismiss={this.toggleConfirmation}
-          />
-        )}
-      </li>
+        <Content blur={showConfirmation}>
+          <Checkbox>{task.status && <IndicatorIcon status={task.status} />}</Checkbox>
+          <StyledLink href={learnMoreUrl} onClick={this.handleClick}>
+            <ItemHeader status={task.status}>{task.title}</ItemHeader>
+          </StyledLink>
+          <Description>{description}</Description>
+          <SkipButton
+            size="xsmall"
+            hide={!showSkipButton}
+            onClick={this.toggleConfirmation}
+          >
+            {t('Skip task')}
+          </SkipButton>
+        </Content>
+        <Confirmation
+          task={task.task}
+          onSkip={() => this.onSkip(task.task)}
+          dismiss={this.toggleConfirmation}
+          hide={!showConfirmation}
+        />
+      </Item>
     );
-  },
-});
+  }
+}
+
+export const animateStripes = keyframes`
+  0% {
+    background-position: 0 0;
+  }
+  100% {
+    background-position: 60px 0;
+  }
+`;
+
+const Item = styled('li')`
+  position: relative;
+  padding: 15px 20px 15px 75px;
+  line-height: 1.2;
+  border-bottom: 1px solid ${p => p.theme.borderLight};
+  background: #fff;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.06);
+  font-size: 14px;
+
+  ${p =>
+    p.status === 'pending' &&
+    css`
+      background-size: 30px 30px;
+      background-image: linear-gradient(
+        135deg,
+        rgba(0, 0, 0, 0.03) 25%,
+        transparent 25%,
+        transparent 50%,
+        rgba(0, 0, 0, 0.03) 50%,
+        rgba(0, 0, 0, 0.03) 75%,
+        transparent 75%,
+        transparent
+      );
+      animation: ${animateStripes} 3s linear infinite;
+    `}
+`;
+
+const Content = styled('div')`
+  position: relative;
+  filter: ${p => p.blur && 'blur(3px)'};
+`;
+
+const Checkbox = styled('div')`
+  height: 44px;
+  width: 44px;
+  background: #fff;
+  border: 3px solid ${p => p.theme.borderDark};
+  border-radius: 46px;
+  position: absolute;
+  top: -5px;
+  left: -58px;
+`;
+
+const indicatorStyles = {
+  skipped: ['icon-close', 'borderDark'],
+  pending: ['icon-ellipsis', 'borderDark'],
+  complete: ['icon-checkmark-sm', 'green'],
+};
+
+const IndicatorIcon = styled(({status, ...props}) => (
+  <InlineSvg {...props} src={indicatorStyles[status][0]} />
+))`
+  position: relative;
+  color: ${p => {
```

Comment:
Can just be `color: ${p => p.theme[indicatorStyles[p.status][1]]};` (just single lining the function body)